lib/miq_expression/field.rb

```diff
@@ -1,8 +1,7 @@
 class MiqExpression::Field < MiqExpression::Target
   REGEX = /
-(?<model_name>([[:upper:]][[:alnum:]]*(::)?)+)
-(?!.*\b(managed|user_tag)\b)
-\.?(?<associations>[a-z][0-9a-z_\.]+)?
+(?<model_name>[[:upper:]][[:alnum:]]+(?:::[[:upper:]][[:alnum:]]+)*)
+(?<associations>\.[a-z][0-9a-z_\.]+)?
```

Comment:
This is slightly wrong - the dot should not be part of the capture group.  I did something similar in my other PR and used:

```
(?<associations>(?<=\.)[a-z][0-9a-z_\.]+)?
```

Reply:
I pulled it out later